From 82b84d66be0ac70501babbfdc82dfedd41eadedf Mon Sep 17 00:00:00 2001
From: Alban Bedel <alban.bedel@avionic-design.de>
Date: Mon, 10 Sep 2018 13:38:34 +0200
Subject: [PATCH 1/4] Disable the depmod config changes, call to depmod and
 initramfs update

All this only make sense when building on the target host.

Signed-off-by: Alban Bedel <alban.bedel@avionic-design.de>
---
 Makefile.real | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/Makefile.real b/Makefile.real
index a0f4916..cddec8e 100644
--- a/Makefile.real
+++ b/Makefile.real
@@ -94,9 +94,6 @@ install: modules
 		modules_install
 	@./scripts/blacklist.sh $(KLIB)/ $(KLIB)/$(KMODDIR)
 	@./scripts/compress_modules.sh $(KLIB)/$(KMODDIR)
-	@./scripts/check_depmod.sh
-	@/sbin/depmod -a
-	@./scripts/update-initramfs.sh $(KLIB)
 	@echo
 	@echo Your backported driver modules should be installed now.
 	@echo Reboot.
@@ -108,8 +105,6 @@ modules_install: install
 .PHONY: uninstall
 uninstall:
 	@./scripts/uninstall.sh
-	@/sbin/depmod -a
-	@./scripts/update-initramfs.sh $(KLIB)
 	@echo
 	@echo Your backported driver modules should be uninstalled now.
 	@echo Reboot.
-- 
2.17.1

